taekwongoFront/components/Login.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component } from 'react';
+
+import {
+    View,
+    Text,
+    TouchableHighlight,
+    Alert,
+    StyleSheet,
+    TextInput
+} from 'react-native';
+
+import {
+    StackNavigator
+} from 'react-navigation';
+
+import SignUp from "./SignUp";
+//import * as LoginConector from '../conectors/LoginConector.js';
+
+export default class Login extends Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+            textUser: '',
+            textPassword: '',
+        };
+
+        //Logic methods
+        this.onLogin = this.onLogin.bind(this)
+        this.signUp = this.signUp.bind(this)
+        this.help = this.help.bind(this)
+    }
+
+    render(){
+        return (
+            <View style={styles.container}>
+                <View style={styles.titlePosition}>
+                    <Text style={styles.title}>TaekwonGo!</Text>
+                </View>
+                <View style={styles.form}>
+                    <View style={styles.borderInput}>
+                        <TextInput
+                            style={styles.input}
+                            onChangeText={(text) => this.setState({textUser: text})}
+                            placeholder={'User'}
+                            value={this.state.textUser}
+                            maxLength={30}
```

Comment:
Deberiamos hacer que matchee la longitud en la base de datos
Lo mismo la password

Reply:
Que tome como maxLength la longitud de ese campo en la base?

Reply:
A que te referis con que matchee l along con la bd?? En el log in no me parecen tan importantes los chequeos, pero si te parece retocar algo hagale

Reply:
claro que sea consistente con el tam maximo de la bd

Reply:
esta bueno para tenerlo en cuenta pero nose de donde podriamos obtener ese maximo. sino habria que harcodearlo en algun lado pero que este en un lugar unico del sistema cuanto va a ser el largo maximo del nombre de usuario

Reply:
Podemos tener validadores que hagan estos chequeos en el backend. En el log in no tiene sentido validar el tamaño, habría que validarlo al momento de registrarse

Reply:
si buen punto